hoodie-client/src/main/java/com/uber/hoodie/io/compact/HoodieRealtimeTableCompactor.java

```diff
@@ -98,20 +98,20 @@ public HoodieCompactionMetadata compact(JavaSparkContext jsc, HoodieWriteConfig
     }
 
     log.info("After filtering, Compacting " + operations + " files");
-    List<CompactionWriteStat> updateStatusMap =
+    List<HoodieCompactionWriteStat> updateStatusMap =
```

Comment:
Okay - Let us not mix that in this diff. Its okay to leave the name as is for now.